lib/meilisearch-rails.rb

```diff
@@ -50,6 +50,7 @@ class IndexSettings
       OPTIONS = %i[
         searchableAttributes
         filterableAttributes
+        sortableAttributes
```

Comment:
Hey @cjilbert504 thanks a lot for your contribution! ❤️ 

Could you provide some simple testing use-case for this option? 🐈

Reply:
Yep, I can do that, thanks for getting back to me.